package.json

```diff
@@ -5,10 +5,12 @@
   "dependencies": {
     "dotenv": "^8.0.0",
     "express": "^4.16.4",
+    "less": "^3.9.0",
```

Comment:
I just wanted to ask, why is Less installed?  For installing it we need to use react-script-eject (and would lose the create-react-app updates) or we would have to use react-app-rewired (and risk breaking the app). I believe you have not used it as yet. So it would be better not to install it, if it is not essential.